Add initial version of gatsby+sensenet guide (#228)
SenseNet/docs.sensenet.com: content/integrations/gatsby/using-gatsby-with-sensenet.md

---
title: "Using Gatsby with sensenet"
metaTitle: "sensenet Integrations - Using Gatsby with sensenet"
metaDescription: "sensenet Integrations - Using Gatsby with sensenet"
---

> sensenet is the single hub for all your content packed with enterprise grade features. In sensenet everything is a content (blog posts, files, users, roles, comments, etc.) delivered the same way through the API, making it super easy to work with any type of data. With sensenet's and Gatsby's flexibility and power you can build any kind of app or website you need.

Reviewer: ```suggestion
sensenet is the single hub for all your content, packed with enterprise grade features. In sensenet everything is a content (blog posts, files, users, roles, comments, etc.) delivered the same way through the API, making it super easy to work with any type of data. With the flexibility and power of sensenet and Gatsby you can build any kind of app or website you need.
```

## When Is sensenet a Good Idea?

**sensenet is great for:**

- Complex project structures with managing large and complex sitemaps
- Teams and projects with complex permission setting requirements
- Storing and delivering data for multiple projects/apps from one repository
- Projects that involve multiple stages of content creation, versioning and approval

> - Storing massive amounts of data

Reviewer: ```suggestion
- Storing massive amounts of data seurely
```

**sensenet is not-so-great for:**

- Teams without any development expertise
- Simple projects with static content that is rarely updated
- Companies that prefer on-premise solutions

## What Does It Take to Implement sensenet?

sensenet is an api-first headless content management platform that makes delivering, managing and maintaining content easy for all devices and platforms.

As a cloud based solution sensenet is hosted and maintained for you in our own cloud environment in fully isolated container infrastructure, which means that with a sensenet repository you get your own secure environment with a custom url, a separate database and a separate API endpoint.

> Cloud-based approach lets you forget about installation and operation tasks, all the subscriber repositories are on the same version and this allows us to deploy new features and patches for free, quickly and efficiently to all of the customer repositories on a regular basis.

Reviewer: ```suggestion
Cloud-based approach lets you forget about installation and operation tasks, all the subscriber repositories are on the same version and this allows us to deploy new features and patches for quickly and efficiently to all of the customer repositories (for free) on a regular basis.
```

Hooking up sensenet to Gatsby takes just a few minutes:

> 1. Create a sensenet account to get a repository

Reviewer: ```suggestion
1. [Create a sensenet account](https://profile.sensenet.com/?redirectToLogin) to get a repository
```
